sdk/test/Services/S3/UnitTests/Custom/General.cs

```diff
@@ -36,8 +36,6 @@ public class GeneralTests
             };
         private static HashSet<string> TypesToIgnore = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
             {
-                "Amazon.S3.Model.Internal.MarshallTransformations.HeadBucketRequestMarshaller",
```

Comment:
You can even remove the `PublicMarshallersTest` now, right?

Reply:
Good point. Removed.

Reply:
You can also remove `NamespacesToAnalyze`, it was only used in that test.